src/java.base/share/classes/sun/security/provider/certpath/OCSP.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2009, 2017, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2009, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright year update not needed for pure backports when the original change doesn't touch it. May cause merge conflicts. Feel free to revert.